vkdgen.py

```diff
@@ -47,47 +47,53 @@
 alias int32_t = int;
 alias int64_t = long;
 
-@nogc pure nothrow {
-	uint VK_MAKE_VERSION(uint major, uint minor, uint patch) {
+@nogc pure nothrow {{

```

Comment:
> The commit message is "Changed flow back into original khronos structure". This has nothing to do with changing how the templates get populated.

On the contrary. Your way to just write each element into types and functions file alphabetically sorted (?!?) lead to unread-ability. With turning back to khronos structure I was able to nicely format the sections.